rllib/models/tests/test_distributions_v2.py

```diff
@@ -71,6 +71,22 @@ def setUpClass(cls) -> None:
         np.random.seed(42)
         torch.manual_seed(42)
 
+    def _check_to_from_dict(self, dist_class, *, sample_input=None):
```

Comment:
Can we rename this to "test_distributions.py" and the old file to "test_action_distributions.py" (also in the file -> "class TestActionDistributions")? This adds more clarity in these messy times :)

Reply:
done.